net/dpdk/Makefile

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := dpdk
+PKG_VERSION := 21.05
+
+PKG_SOURCE := dpdk-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL := http://static.dpdk.org/rel
+PKG_HASH := 1e12499b4c5fcdb57c83e5fe184ea6becddf7cf092893c2c5ef2efb0eec12f0b
+
+PKG_BUILD_DEPENDS:= meson/host ninja/host
```

Comment:
I'm unsure if `ninja/host` is still required. @neheb ?

Reply:
It is not. Just meson/host.

Reply:
Thanks @PolynomialDivision @neheb ..I'll remove `ninja/host`